mne/epochs.py

```diff
@@ -402,6 +409,8 @@ def decimate(self, decim, copy=False):
         else:
             epochs._decim_slice = decim_slice
             epochs.times = epochs._raw_times[epochs._decim_slice]
+        epochs.tmin = epochs.times[0]
+        epochs.tmax = epochs.times[-1]
```

Comment:
Before, since unchanged, tmin and tmax were related to the original tmin and tmax.
But if the number of sample before decimation is not a multiple of the decimation rate + 1, tmax before and after decimation will not be the same, and this will raise an error in the \__init\__() function of the class epochs as explained in my first comment of this PR with a detailed exemple (see above).